app/views/content/steps-to-become-a-teacher/_step_1_decide_who_to_teach.html.erb

```diff
@@ -7,4 +7,6 @@
   <li><a href="/become-a-further-education-teacher">further education</a> (for example, teaching at further education colleges or sixth form colleges)</li>
 </ul>
 
-<p>Follow these steps if you want to teach primary or secondary, including <a href="/teach-disabled-pupils-and-pupils-with-special-educational-needs">teaching disabled pupils and pupils with special educational needs</a> in these settings.<p>
+<p>Follow these steps if you want to teach primary or secondary, including <a href="/teach-disabled-pupils-and-pupils-with-special-educational-needs">teaching disabled pupils and pupils with special educational needs</a> in these settings.</p>
+
+<p>If you’re an undergraduate, interested in teaching chemistry, computing, languages, maths or physics then you could do a <a href="/teaching-internship-providers">paid teaching internship</a> to find out if teaching is right for you.</p>
```

Comment:
```suggestion
<p>If you’re an undergraduate and interested in teaching chemistry, computing, languages, maths or physics then you could do a <a href="/teaching-internship-providers">paid teaching internship</a> to find out if teaching is right for you.</p>
```

not sure about the first comma in terms of readability @philspray? Is 'and' easier to understand?

Reply:
Yep I agree, and is better